src/pages/docs/deployments/bulk-deployment-creation.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-09-13
+modDate: 2023-09-13
+title: Bulk deployment creation
+description: Octopus Deploy can use bulk deployment creation when deploying to multiple environments or tenants
+navOrder: 160
+---
+
+:::div{.hint}
+Bulk deployment creation will be available from Octopus Deploy **2023.4**
+:::
+
+When deploying a release to multiple environments or tenants through the Octopus Portal, the bulk deployment creation feature will immediately take you to a new server task which will create a deployment for each environment or tenant you are deploying to.
+
+In the bulk deployment creation server task, you can see the progress of deployments being created and any errors during creation.
+
+:::figure
+![](/docs/deployments/bulk-deployment-creation.png)
+:::
+
+You can also navigate away from the server task and it will continue creating deployments in the background.
+
+If you need to return to a bulk deployment creation server task, you can go to the **Tasks** page and filter down to **Bulk deployment creation** task types.
+
+# Retrying deployment creation failures
+If a deployment to an environment or tenant fails to create, you can **Re-run** the bulk deployment creation server task. This will create a new server task that reattempts creating deployments for the environments or tenants that failed.
+
+:::figure
+![](/docs/deployments/bulk-deployment-creation-retry.png)
+:::
+
+# Permissions
+To create, view or re-run a bulk deployment creation server task, your permissions will need to be scoped to at least all the environments and tenants that a bulk deployment creation server task is creating deployments for.
```

Comment:
Consider rewording to avoid the passive voice:

Permissions will need to be scoped to all requested environments and tenants when creating, viewing, or re-running a bulk deployment.

Reply:
I feel "requested" could be ambiguous though as it's not clear what requested is referring to.

Reply:
```suggestion
Permissions will need to be scoped to all requested environments and tenants when creating, viewing, or re-running a bulk deployment server task.
```